critiquebrainz/frontend/views/mapping.py

```diff
@@ -147,12 +147,11 @@ def spotify_report():
         flash.success(gettext("Incorrect Spotify mapping has been reported. Thank you!"))
         return redirect(url_for('.spotify_list', release_group_id=release_group_id))
 
-    else:
-        try:
-            album = spotify_api.get_album(spotify_id)
-        except ExternalServiceException:
-            flash.error(gettext("You need to specify existing album from Spotify!"))
-            return redirect(url_for('.spotify_list', release_group_id=release_group_id))
+    try:
+        album = spotify_api.get_album(spotify_id)
+    except ExternalServiceException:
+        flash.error(gettext("You need to specify existing album from Spotify!"))
+        return redirect(url_for('.spotify_list', release_group_id=release_group_id))
```

Comment:
I was getting https://gist.github.com/ferbncode/307bc3f3e13f5d65428aad6180ace1ed error when running pylint. I have undone this change now as it may be done separately.